services/openvpn/service/factory.go

```diff
@@ -92,6 +94,9 @@ You should probably need to do port forwarding on your router: %s:%v -> %s:%v.`,
 			)
 		},
 
+		proposalFactory: func(currentLocation dto_discovery.Location) dto_discovery.ServiceProposal {
```

Comment:
We could start defining non-anonymous functions here to reduce factory bloating and improve readability.

Reply:
Yes, I have it in yet another PR. Dont want to grow this one